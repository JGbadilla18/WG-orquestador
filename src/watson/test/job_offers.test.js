const testWatson = require('./testWatson');

watsonText = 'Oportunidad laboral.'

it(watsonText, () => {
  expect.assertions(1);
  return testWatson(watsonText).then(data => expect(data).toMatchSnapshot());
});
