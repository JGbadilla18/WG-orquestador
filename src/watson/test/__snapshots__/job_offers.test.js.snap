// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Oportunidad laboral. 1`] = `
Object {
  "actions": Array [],
  "components": Array [
    Object {
      "response_type": "text",
      "text": "Nos alegra que te quieras unir a nuestro equipo. Para aplicar, envía tu CV al correo electrónico oportunidades@bi.com.gt o acércate a la oficina de Recursos Humanos ubicada en 7a avenida 11-15 zona 1 de la Ciudad de Guatemala.",
    },
    Object {
      "response_type": "pause",
      "time": 10000,
      "typing": true,
    },
  ],
}
`;
